src/main/webapp/app/exercises/shared/result/result.service.ts

```diff
@@ -24,6 +24,12 @@ export type EntityResponseType = HttpResponse<Result>;
 export type EntityArrayResponseType = HttpResponse<Result[]>;
 export type ResultsWithPointsArrayResponseType = HttpResponse<ResultWithPointsPerGradingCriterion[]>;
 
+export interface Badge {
+    class: string;
```

Comment:
Could you use another name? It clashes with "class" from TS

Reply:
was that way before and I would prefer not to make any more changes